### DISCUSSION TITLE
Audio Jack win11

### DISCUSSION THREAD

**Discussion by gubbirk** [GrandOrgue/grandorgue#2081] [Help]
When I plug in headphone 3,5 jack, the sound still comes from internal pc speakers. 

Other apps works ok.

**Comment by gubbirk** [comment]
My pc is dell vostro 3520

**Comment by oleg68** [comment]
Check File->Settings->Audio: what audio device is selected?

**Reply by gubbirk** [reply]
Audio device selected:
Cirrus logic high definition audio

**Comment by gubbirk** [comment]
By the way, I don’t find the path you mentioned. I use the path:
Audio/midi /  audio/midi settings / audio output

**Comment by gubbirk** [comment]
My problem is not solved yet.

I have tried Bluetooth, but too much latency, so I depend on jack.

**Comment by gubbirk** [comment]
I have made progress

it is possible to choose between speaker and headphone in the config. So when I choose headphones it works!

I expected that the program would change automatically to headphones when headphones were plugged in.

When I unplug the headphones the program crashes.

**Reply by oleg68** [reply]
> When I unplug the headphones the program crashes.

What version of GrandOrgue are you using?

> I expected that the program would change automatically to headphones when headphones were plugged in.

You have to choose something as "default".

**Comment by oleg68** [comment]
> Audio/midi / audio/midi settings / audio output

Seems your version of GrandOrgue is too old. Download one from https://github.com/GrandOrgue/grandorgue/releases/tag/3.15.4-1

**Reply by gubbirk** [reply]
I get warnings from pc when downloading from GitHub. 

I am also not sure which file to download.
I tried “grand….3.15.4-1.windows.x86_64.exe”

okey, I downloaded from sourceforge, thus the old version

I have now successfully installed ver 3.15.4_1 from github


But I must still go to settings to manually choose between internal speakers and headphones. Perhaps this is intended?

At least the program doesn’t crash

**Reply by oleg68** [reply]
> But I must still go to settings to manually choose between internal speakers and headphones. Perhaps this is intended?

Post the screenshot of the "Change Device" window here. Maybe there is an appropriate choice.

**Reply by gubbirk** [reply]
![image](https://github.com/user-attachments/assets/7b7bcdc8-d5d6-4c65-86da-e5e148d10eac)

**Comment by oleg68** [comment]
Try to use "PortAudio: MME: Microsoft Sound Mapper - Output", "PortAudio: Windows DirectSound: Primary Sound Driver" or "RtAudio: ds: Primary Sound Driver"

**Reply by gubbirk** [reply]
The two first worked well, the third made the program crash.
Well, then I have to remember these two options.

Thank you very much for your support!
